Makefile

```diff
@@ -51,6 +51,7 @@ vendor:
 	go install github.com/golang/mock/mockgen
 	go install tools/nextimportslint.go
 	go install github.com/google/wire/cmd/wire
+	go install github.com/skygeario/openapi3-gen/cmd/openapi3-gen
```

Comment:
Could you also add `github.com/google/wire/cmd/wire` and `github.com/skygeario/openapi3-gen/cmd/openapi3-gen` to `devtools/devtools.go` so that the exact version is recorded in `go.mod`?